en/index.html

```diff
@@ -25,26 +25,54 @@
  <body style="direction: ltr;">
     <div id="fb-root"></div>
 
-    <div id="header">
-      <div class="hgroup">
-        <h1>August Penguin 2016</h1>
+    <div class="container">
+      <header>
+        <nav>
+          <ul>
+            <li>
+              <a href="index.html">About</a>
+            </li>
+            <li>
+              <a href="#schedule">Agenda</a>
+            </li>
+            <li>
+              <a href="code_of_conduct.html">Code of Conduct</a>
```

Comment:
we should also add the header in the code of conduct or open the code of conduct in a separate tab